drivers/gpio/Kconfig.litex

```diff
@@ -8,3 +8,9 @@ config GPIO_LITEX
 	depends on SOC_RISCV32_LITEX_VEXRISCV
 	help
 	  Enable Litex GPIO driver.
+
+config GPIO_LITEX_IRQ
```

Comment:
On other SoCs we generally don't have Kconfig options to enable IRQ support in GPIO driver. The savings are minimal. Also, the option is easy to overlook by the user. Maybe we should remove it? I.e. the IRQ support should be always enabled.

Reply:
I did get rid of that option so GPIO IRQs are always enabled right now.